web/themes/joinup/joinup_theme.theme

```diff
@@ -19,13 +19,77 @@ use Drupal\file\Entity\File;
 use Drupal\image\Entity\ImageStyle;
 
 /**
+ * Check if route is special.
+ *
+ * @param \Drupal\Core\Routing\RouteMatchInterface $route_match
+ *   The route object.
+ *
+ * @return array
+ *   The route type.
+ */
+function joinup_check_route(RouteMatchInterface $route_match) {
+  $route_name = $route_match->getRouteName();
+
+  // Declare available types.
+  $types = [
+    'create' => FALSE,
+    'tabs' => FALSE,
+  ];
+
+  // Check if page has a create form.
+  $create_content_routes = [
+    'collection.propose_form',
+    'rdf_entity.propose_form',
+  ];
+
+  // Check if string has 'add' on the end.
+  if (preg_match('/.add$/', $route_name)) {
+    $types['create'] = TRUE;
+  }
+
+  if (!empty($route_name) && in_array($route_name, $create_content_routes)) {
+    $types['create'] = TRUE;
+  }
+
+  // Check if page has form vertical tabs.
+  $tabs_routes = [
+    'entity.rdf_entity.edit_form',
+    'rdf_entity.propose_form',
+    'solution.collection_solution.add',
+    'rdf_entity.rdf_add',
+  ];
+
+  if (!empty($route_name) && in_array($route_name, $tabs_routes)) {
+    // Get entity type on edit form.
+    $entity = $route_match->getParameter('rdf_entity');
+    if (!empty($entity)) {
+      $rdf_type = $entity->getType();
+    }
+    else {
+      // Get entity type on create form.
+      $rdf_type = $route_match->getParameter('rdf_type');
```

Comment:
There are two ifs, because on create pages like /rdf_entity/add/collection entity variable is empty, so we need to check the parameter.